pkg/probes/dummy/dummy.go

```diff
@@ -0,0 +1,38 @@
+package dummy
+
+import (
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type DummyProbe struct{}
+
+const (
+	startingToken = "DUMMY_START"
+	endingToken   = "DUMMY_END"
+)
+
+// GetStartingToken returns the string token used to signal the beginning of the probe's output
+func (prb DummyProbe) GetStartingToken() string { return startingToken }
+
+// GetEndingToken returns the string token used to signal the end of the probe's output
+func (prb DummyProbe) GetEndingToken() string { return endingToken }
+
+// GetMachineImageID returns the string ID of the VM image to be used for the probe instance
+func (prb DummyProbe) GetMachineImageID(platformType string, cpuArch string, region string) (string, error) {
```

Comment:
My personal preference is to delete the names of unused function params (here, line 26, and line 37). That makes it extremely clear to anyone looking at the code that we are strictly providing the parameters to satisfy an interface, but we don't care about the values themselves.
```suggestion
func (prb DummyProbe) GetMachineImageID(string, string, string) (string, error) {
```

Reply:
Is there an alternative to this?
```
pkg/verifier/gcp/entry_point.go:58:14: cannot use dummy.Probe{} (value of type dummy.Probe) as probes.Probe value in assignment: dummy.Probe does not implement probes.Probe (wrong type for method GetMachineImageID)
		have GetMachineImageID(string, string, string) (string, error)
		want GetMachineImageID(string, "github.com/openshift/osd-network-verifier/pkg/data/cpu".Architecture, string) (string, error)

Reply:
That error looks... odd. Almost like a bug in our linter. Let's dig into this a bit more.